packages/grafana-ui/src/components/BarGauge/BarGauge.tsx

```diff
@@ -567,11 +567,12 @@ export function getBarGradient(props: Props, maxSize: number): string {
  * Only exported to for unit test
  */
 export function getValueColor(props: Props): string {
-  const { theme, value } = props;
+  const { value } = props;
   if (value.color) {
     return value.color;
   }
-  return getColorFromHexRgbOrName('gray', theme.type);
+
+  return 'gray';
```

Comment:
nit: could these "magic" color strings be part of a enum or const?

Reply:
fixed :)